### DISCUSSION TITLE
Is there a way to change the boarder?

### DISCUSSION THREAD

**Discussion by AndOrangutan** [williamboman/nvim-lsp-installer#531] [Q&A] [answered]
Is there any way that I could change the border style used, I prefer `none` or `solid`?
![image](https://user-images.githubusercontent.com/40435989/157349027-e5925cde-45ca-41d6-93f2-f2e12093eb56.png)

**Comment by williamboman** [comment, marked as answer]
You can utilize the `FileType` autocommand for this:

```vim
autocmd FileType lsp-installer lua vim.api.nvim_win_set_config(0, { border = "none" })
```

See `:h nvim_win_set_config` for which border options there are.
